CLI/test/ressources/checkMissingCopies_2/output_expected/error.txt

```diff
@@ -0,0 +1,4 @@
+ReturnCode=MissingCopy_
+2 errors in "input\MissingCopy.rdz.tcbl":
+Line 5[8,18] <46, Error, Directives> - Dependent COPY COPY3 not found
```

Comment:
There is no indication that it's **COPY2** which need COPY3 and COPY4?

Reply:
As I explained it in this PR description, I could not find a simple way to keep the info.
When we first detect it, we know it's COPY2 that needs COPY3 (and COPY4) but with the current code this info is lost for upper levels. To keep it, we would have to change the `List<CopyDirective> missingCopies` into something that holds the link between the missing copy and its including copy (if any). this has impacts on several classes/methods.

Reply:
You know have the choice with either this solution or #1815 which supports parent copy in message.